package.json

```diff
@@ -3,6 +3,10 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
+        "@fortawesome/fontawesome-free": "^5.15.4",
```

Comment:
is it really necessary to add 4 packages to get an icon for hamburger menu?

Reply:
Added Font Awesome CDN embed code in index.html instead of the previous solution.